src/app/components/normal-component/online-game-selection/online-game-selection.component.ts

```diff
@@ -9,12 +9,12 @@ export class OnlineGameSelectionComponent {
 
     public selectedGame: string;
 
-    public constructor(private gameService: GameService) {
+    public constructor(private readonly router: Router) {
     }
     public pickGame(pickedGame: string): void {
         this.selectedGame = pickedGame;
     }
     public async createGame(): Promise<void> {
```

Comment:
say what you do, do what you say: il faut renommer createGame à présent!

Reply:
renommé (navigateToOnlineGameCreation), si t'as une meilleure idée de nom dis moi !